src/Apps/Legion/Components/Diagram/Diagram.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { getDebugLogger } from '../../../../Models/Services/Utils';
+import { IDiagramProps, TDiagramAnnotationPlacement } from './Diagram.types';
+import { useClassNames } from './Diagram.styles';
+import { Image, Spinner, SpinnerSize } from '@fluentui/react';
```

Comment:
Spinner and Image are available on Fluent 9, would be a good idea to see if those work here.